.gitmodules

```diff
@@ -1,6 +1,3 @@
-[submodule "gvpp"]
```

Comment:
Hahah, for the second time. Well, now we know what does not reviewing lead to :stuck_out_tongue:

Reply:
Yes we do :smile: .